syfertext/token.py

```diff
@@ -48,6 +56,27 @@ def text(self):
         """Get the token text"""
         return str(self.doc.vocab.store[self.orth])
 
+    @property
+    def text_(self):
+        """Get the token text in Syft's String type"""
+        return String(self.text, owner=self.owner)
```

Comment:
This does not need to by a String any more, you can just return a `str` type. (please change the docstring to reflect this too.

Also, there should not be too methods. self.text_ and self.text, only self.text_. The method self.text. just remove self.text and rename self.text_ to self.orth_

Reply:
regarding removal `self.text` property, it is being used on a lot of other places and if I remove it, it will break a lot of code. So I just renamed `self.text_` to `self.orth_` and updated the doc string.